webapp/src/components/ChatUserCard.vue

```diff
@@ -0,0 +1,112 @@
+<template lang="pug">
+.c-chat-user-card
+	.g-background-blocker(v-if="!userAction", @click="$emit('close')")
+	.user-card(v-if="!userAction", ref="card", @mousedown="showMoreActions=false")
+		avatar(:user="sender", :size="128")
+		.name {{ sender.profile ? sender.profile.display_name : this.message.sender }}
+		.state {{ userStates.join(', ') }}
+		.actions(v-if="sender.id !== user.id")
+			bunt-button.btn-dm(v-if="hasPermission('world:chat.direct')", @click="openDM") message
+			bunt-button.btn-call(v-if="hasPermission('world:chat.direct')", @click="startCall") call
+			menu-dropdown(v-if="$features.enabled('chat-moderation') && (hasPermission('room:chat.moderate') || message.sender === user.id)", v-model="showMoreActions", :blockBackground="false", @mousedown.native.stop="")
```

Comment:
This caused issues for me since ``message`` does not exist. Probably also affects line 6.